src/Scene/Node.js

```diff
@@ -21,7 +21,7 @@ function Node() {
     this.screenSpaceError = 0.0;
     this.loaded = false;
     this.pendingSubdivision = false;
```

Comment:
Yes, but it requires quite a lot of refactoring because `pendingSubdivision` is used in several places that doesn't know about layers (eg TileMesh, Node), while `layerUpdateState` needs a layer.

I'll probably do it later though.